docs/docs/examples/vector_stores/AzureAISearchIndexDemo.ipynb

```diff
@@ -569,36 +569,56 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 37,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Score: 0.8748552\n",
-      "File Name: Unknown\n",
-      "Id: bae0df75-ff37-4725-b659-b9fd8bf2ef3c\n",
+      "Score: 0.80721414\n",
+      "File Name: paul_graham_essay.txt\n",
+      "Id: 157c7095-654c-4bd9-9620-1aa8453cae80\n",
       "\n",
       "Extracted Content:\n",
-      "Inception\n",
+      "[4] You can of course paint people like still lives if you want to, and they're willing. That sort of portrait is arguably the apex of still life painting, though the long sitting does tend to produce pained expressions in the sitters.\n",
```

Comment:
This response looks very different from the previous one. How much of the different can be attributed to the retrieval change versus the fact that I assume the previous version didn't use gpt-4o? What's with the [<number>] prefixes? Is that format asked for somewhere in the prompt?

Reply:
I've reverted back to the default settings using "gpt-35-turbo". which is the default value for the llamaindex AzureOpenAI LLM class (and ada002 for embeddings respectively). 
Answer is identical to the previous.

The [] is actually in the raw txt file: https://github.com/run-llama/llama_index/blob/main/docs/docs/examples/data/paul_graham/paul_graham_essay.txt